presentation-compiler/src/main/dotty/tools/pc/PcInlayHintsProvider.scala

```diff
@@ -118,19 +123,18 @@ class PcInlayHintsProvider(
           adjustPos(pos).endPos.toLsp,
           label,
           InlayHintKind.Type,
+          InlayHintOrigin.TypeParameters
         )
       case InferredType(tpe, pos, defTree)
           if !isErrorTpe(tpe) =>
         val adjustedPos = adjustPos(pos).endPos
-        if firstPassHints.containsDef(adjustedPos.start) then firstPassHints
-        else
-          firstPassHints
-            .add(
-              adjustedPos.toLsp,
-              LabelPart(": ") :: toLabelParts(tpe, pos),
-              InlayHintKind.Type,
-            )
-            .addDefinition(adjustedPos.start)
```

Comment:
It was removed out of sudden in Scala 2 presentation compiler, I believe it is no longer necessary as we're deduplicating inlay hints during `add`